spec/brcobranca/banco_mercantil_lote_spec.rb

```diff
@@ -0,0 +1,69 @@
+# -*- encoding: utf-8 -*-
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+
+describe Brcobranca::Boleto::MercantilLote do
+  before(:each) do
+    @valid_attributes = {
+      :especie_documento => "DM",
+      :moeda => "9",
+      :data_documento => Date.today,
+      :dias_vencimento => 1,
+      :aceite => "S",
+      :quantidade => 1,
+      :valor => 0.0,
+      :local_pagamento => "QUALQUER BANCO ATÉ O VENCIMENTO",
+      :cedente => "Kivanio Barbosa",
+      :documento_cedente => "12345678912",
+      :sacado => "Claudio Pozzebom",
+      :sacado_documento => "12345678900",
+      :agencia => "4042",
+      :conta_corrente => "61900",
+      :convenio => 12387989,
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Separate every 3 digits in the integer portion of a number with underscores(_).